oauth2/oauth2-core/src/main/java/org/springframework/security/oauth2/core/http/converter/OAuth2AccessTokenResponseHttpMessageConverter.java

```diff
@@ -129,7 +129,7 @@ public final void setTokenResponseParametersConverter(Converter<OAuth2AccessToke
 	 * A {@link Converter} that converts the provided
 	 * OAuth 2.0 Access Token Response parameters to an {@link OAuth2AccessTokenResponse}.
 	 */
-	private static class OAuth2AccessTokenResponseConverter implements Converter<Map<String, String>, OAuth2AccessTokenResponse> {
+	public static class OAuth2AccessTokenResponseConverter implements Converter<Map<String, String>, OAuth2AccessTokenResponse> {
```

Comment:
Let's please make these classes `final`, now that they are public. Spring Security's public API typically starts out as `final`.